includes/definitions/discovery/intracom-2wcx.yaml

```diff
@@ -0,0 +1,503 @@
+---
+mib: PTP-MODEM-MIB:ICOM-ETHER-MIB:ICOM-ENTITY-SENSOR-EXT-MIB:ICOM-MODEM-PERF-MIB:ICOM-PERFHIST-TC-MIB
+modules:
+    os:
+        sysDescr_regex: '/5\d(?<hardware>.*?)( Wireless|,).*Version\s+(?<version>[\d-]+)/'
```

Comment:
this doesnt work since the info is not in the sysdescr, but there seems to be other oids for version and hardware